### PR TITLE
Fixed #28008 -- Removed getElementsByClassName() in debug view template.

### PR DESCRIPTION
**Pull request by abhishekg1128**
I have used querySelectorAll to replace getElementsByClassName().
https://code.djangoproject.com/ticket/28008

### CODE REVIEW

**Comment by timgraham** [comment]
Please rebase your branch after ea542a9c7239b5b665797b7c809f1aceb0b412cf. The obsolete code should be removed rather than commented out.

**Comment by abhishekg1128** [comment]
Done.

File: django/views/templates/technical_500.html [hunk 1]
```diff
@@ -65,33 +65,17 @@
...
+     for (var e = 0; e < elems.length; e++) {
+       elems[e].style.display = 'none';
+     }
+   }
+   window.onload = function() {
+     hideAll(document.querySelectorAll('table.vars'));
+     hideAll(document.querySelectorAll('ol.pre-context'));
+     hideAll(document.querySelectorAll('ol.post-context'));
+     hideAll(document.querySelectorAll('div.pastebin'));
+   }
```
Comment (claudep) [inline django/views/templates/technical_500.html]: Please respect indentation of other JS functions in the file.

**Comment by claudep** [comment]
Do you have a specific reason to change the existing code to 3-space indents instead of using 4-space indents like it was before?

**Comment by abhishekg1128** [comment]
Earlier you said to respect indentation of other JS functions in the file. I think if make all indentations same then it will be fine. Not a problem I will change indentation to 4 spaces.

**Comment by claudep** [comment]
Sorry, I was not clear enough. Now it looks better!

**Pull request merged**